workflow/snakemake_rules/main_workflow.smk

```diff
@@ -103,7 +103,7 @@ rule align:
             --output-basename {params.basename} \
             --output-fasta {params.uncompressed_alignment} \
             --output-insertions {output.insertions} > {log} 2>&1;
-        xz -2 {params.uncompressed_alignment};
+        xz -2 -T {threads} {params.uncompressed_alignment};
         xz -2 {params.outdir}/{params.basename}*.fasta
```

Comment:
Surely we'd want to use `-T {threads}` for the immediately-following `xz` invocation too?

Reply:
I tested that, too, and will post the benchmark results tomorrow. (For good measure, I also tested a command based on `xargs`, and the improvement wasn't as stark as just multithreaded `xz`.)

Reply:
Awesome, thanks!